src/compiler/compile/render_ssr/handlers/Element.ts

```diff
@@ -69,10 +81,12 @@ export default function(node: Element, renderer: Renderer, options: RenderOption
 			}
 		});
 
-		renderer.add_expression(x`@spread([${args}], ${class_expression})`);
+		renderer.add_expression(x`@spread([${args}], { classes: ${class_expression}, styles: ${style_expression} })`);
```

Comment:
the `spread` internal ssr helper now takes an object with `styles` and `classes`